src/lib/QtWidgets/QTabWidget.ts

```diff
@@ -71,13 +69,11 @@ export class QTabWidget extends QWidget<QTabWidgetSignals> {
      * @returns The new index of the tab
      */
     insertTab(index: number, page: QWidget, iconOrLabel: QIcon | string | undefined, label: string): number {
-        let newIndex
+        let newIndex;
         if (iconOrLabel instanceof String) {
             newIndex = this.native.insertTab2(index, page.native, iconOrLabel);
-        } else if (iconOrLabel instanceof QIcon && label instanceof String) {
-            newIndex = this.native.insertTab2(index, page.native, label);            
-        } else {
-            newIndex = this.native.insertTab(index, page.native, iconOrLabel.native, label);
+        } else if (iconOrLabel instanceof QIcon && label !== undefined) {
+            newIndex = this.native.insertTab2(index, page.native, iconOrLabel.native, label);
```

Comment:
This should be `.insertTab()` shouldn't it? You're passing the icon to the newly written function that doesnt accept it

Reply:
I've might be right. I'll changed the names to indicate the number of arguments otherwise I'll never get it straight.